src/config.js

```diff
@@ -9,6 +9,7 @@ exports.getConfig = function() {
       username: process.env.OPENHIM_USERNAME || 'root@openhim.org',
       password: process.env.OPENHIM_PASSWORD || 'openhim-password',
       trustSelfSigned: process.env.TRUST_SELF_SIGNED === 'true'
-    })
+    }),
+    acceptNullValues: !(process.env.ACCEPT_NULL_VALUES === 'false')
```

Comment:
@bradsawadye This syntax is confusing. If you need the boolean flipped make sure the variable name makes what you expect. You could change the name of the config field name to nullable then the boolean flip makes sense

Reply:
addressed in https://github.com/jembi/openhim-mediator-mapping/pull/21/commits/4a37f5f33a74753cc5a9f11705407cb9b974570e